tests/apps/lims/test_limsjson.py

```diff
@@ -74,7 +73,8 @@ def test_parsing_rml_json(rml_order_to_submit: dict) -> None:
     assert sample_data["name"] == "sample3"
     assert sample_data["pool"] == "pool-2"
     assert sample_data["application"] == "RMLS05R150"
-    assert sample_data["data_analysis"] == "fluffy"
+    assert sample_data["data_analysis"] == str(Pipeline.FLUFFY)
```

Comment:
yes, it is, seems more intuitive to move it to case level. I think the organisation is like this because we don't have a case level for all samples. Since we have introduced the case level for all samples in the backend now we could change this in the orderportal interface, but maybe not in this PR :)